src/artm/regularizer_sandbox/decorrelator_phi.cc

```diff
@@ -17,31 +18,11 @@ bool DecorrelatorPhi::RegularizePhi(::artm::core::Regularizable* topic_model, do
   // read the parameters from config and control their correctness
   const int topic_size = topic_model->topic_size();
 
-  auto topic_name = topic_model->topic_name();
-  std::vector<bool> topics_to_regularize;
-  if (config_.topic_name_size() > 0) {
-    for (int i = 0; i < topic_size; ++i)
-      topics_to_regularize.push_back(false);
-
-    for (int topic_id = 0; topic_id < config_.topic_name_size(); ++topic_id) {
-      for (int real_topic_id = 0; real_topic_id < topic_size; ++real_topic_id) {
-        if (topic_name.Get(real_topic_id) == config_.topic_name(topic_id)) {
-          topics_to_regularize[real_topic_id] = true;
-          break;
-        }
-      }
-    }
-  } else {
-    for (int i = 0; i < topic_size; ++i)
-      topics_to_regularize.push_back(true);
-  }
+  std::vector<bool> topics_to_regularize = core::is_member(config_.topic_name(),
```

Comment:
Now I see why is_member returns "true" by default... I think this is still beter to return "false" by default, because this is consistent with matlab.

std::vector<bool> topics_to_regularize;
if (config_.topic_name().empty())
   topics_to_regularize.assign(config_.topic_name().size(), true)
else
  topics_to_regularize = core::is_meber(config_.topic_name());

This is more verbose, but then you explicitly see how we treat empty topic_name.